### DISCUSSION TITLE
Inbucket integrated into Supabase Studio

### DISCUSSION THREAD

**Discussion by probablykasper** [supabase/supabase#11795] [General]
# Feature request

## Is your feature request related to a problem? Please describe.

With multiple Supabase projects, it can be hard to keep track of what port is for what. You have the API URL, DB URL, Studio URL and Inbucket URL. I often want to access Studio and Inbucket, but don't remember what port to access.

## Describe the solution you'd like

Have Inbucket integrated with Supabase Studio so you have one less URL/port to keep track of. I'd love to have Studio with Inbucket on a port like `54320`, and then I could use `54330` etc for others, no problem remembering that

**Comment by probablykasper** [comment]
A potential alternative solution to this would be to reuse Supabase Studio and Inbucket instances for multiple projects

**Comment by kiwicopple** [comment]
The modularity  of our current setup is great, although I agree it diminishes the DX. Your idea to integrate it is definitely interesting to help with local development, but it's a considerable amount of work to re-create everything that inbucket has done inside Studio. For now, I'll convert this to a feature request in our Discusssions and we can track the popularity of the upvotes

**Reply by probablykasper** [reply]
One pretty simple thing that could be done is to just have a link to Inbucket inside Studio
